modules/project/variables.tf

```diff
@@ -162,6 +162,12 @@ variable "project_create" {
   default     = true
 }
 
+variable "project_skip_delete" {
```

Comment:
We're already in the project module, so the "project" prefix seems redundant. Can we call it just `skip_delete`?